### DISCUSSION TITLE
What do commit colors mean?

### DISCUSSION THREAD

**Discussion by taep96** [jesseduffield/lazygit#1622] [Q&A] [answered]
What do the colors and text after them mean?
![image](https://user-images.githubusercontent.com/64481039/145462903-18005436-89bb-4b9b-abf4-15e07a8832a2.png)

**Comment by jesseduffield** [comment, marked as answer]
green means it's on the master branch. Yellow means it's not on the master branch, and red means it hasn't yet been pushed. This should go into a FAQ somewhere

**Reply by jesseduffield** [reply]
I've updated the FAQ in the readme
